### ISSUE TITLE
Big space above text in OneNote

### ISSUE THREAD

**Issue by jimgust** [JetBrains/JetBrainsMono#210]
I noticed that in OneNote when I use t either the mono or NL font that there is a big space above the text.
![FontTest-OneNote](https://user-images.githubusercontent.com/891903/79567841-80978300-807a-11ea-8699-97325a11787f.png)

**Comment by philippnurullin** [comment]
@jimgust Hi, can you test for this issue v1.0.4 & 1.0.5. Do they return different results?

**Comment by jimgust** [comment]
I uninstalled the font and tried with both v1.0.4 and v1.0.5.  Both had the same issue.  I also noticed a weired height truncation when restarting OneNote.

v.1.0.4
![OneNote_JBMono_v1 0 4](https://user-images.githubusercontent.com/891903/79751003-7cb46c80-82d7-11ea-8626-e39b994742be.png)

v.1.0.5
![OneNote_JBMono_v1 0 5](https://user-images.githubusercontent.com/891903/79751044-8b028880-82d7-11ea-8224-505397c6835b.png)
